src/sdk/bucketspace_fs.js

```diff
@@ -121,11 +122,22 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             };
 
             bucket.name = new SensitiveString(bucket.name);
-            bucket.bucket_owner = new SensitiveString(bucket.bucket_owner);
+            // Backward compatibility will take longer without bucket owner name
```

Comment:
Why do you add this comment "will take longer without bucket owner name"?
There is no other option that we can use...

Reply:
there is, but it might cause a mistake, I thought of using the old bucket_owner property, read the account config file and check if it has the same id...

Reply:
I didn't understand this yet.